project.clj

```diff
@@ -69,7 +69,8 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty9-version]
-                 [prismatic/schema "0.2.2"]
+                 [prismatic/schema "0.4.0"]
+                 [prismatic/plumbing "0.4.2"]
```

Comment:
TK right now (as in the version we use) depends on plumbing 0.2.1, which is not compatible with schema 0.4.0. So if we don't include plumbing this fails. We could I think ... wait for a TK release, bump to that and probably avoid this extra requirement if so desired.